demos/python_demos/multi_camera_multi_person_tracking/utils/ie_tools.py

```diff
@@ -59,12 +59,12 @@ def get_input_shape(self):
         return self.inputs_info[self.input_key]
 
 
-def load_ie_model(model_xml, device, plugin_dir, cpu_extension='', num_reqs=1):
+def load_ie_model(ie, model_xml, device, plugin_dir, cpu_extension='', num_reqs=1):
     """Loads a model in the Inference Engine format"""
     model_bin = os.path.splitext(model_xml)[0] + ".bin"
     # Plugin initialization for specified device and load extensions library if specified
     log.info("Creating Inference Engine")
```

Comment:
This log message should probably be moved together with the `ie` assignment.

Reply:
My understanding is that this log is applicable to lines following after that. It was noted as specific device init, which is still left in place.

Reply:
I don't think the lines below can reasonably be described as "Creating Inference Engine", though. So keep the log entry here if you want, but then edit it to match what actually happens.

Reply:
Re-phrased log in original location. Added log in ie instance creation.